config/alacritty/windows.toml

```diff
@@ -0,0 +1,8 @@
+import = ["/home/kachick/.config/alacritty/common.toml"]
+
+[shell]
+program = "foo"
```

Comment:
Testing code... 🙈

Reply:
Fixed in https://github.com/kachick/dotfiles/commit/bccf604742ee016e4ae680423ae8136b4b2cf1f8